src/main/java/org/ow2/proactive/catalog/service/BucketService.java

```diff
@@ -168,4 +170,19 @@ private BucketEntity findBucketByNameAndCheck(String bucketName) {
         }
         return bucketEntity;
     }
+
+    public List<BucketMetadata> getBucketsByGroups(String sessionId, String ownerName, Optional<String> kind,
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused method parameter "sessionId". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1172)

Reply:
removed